src/Symfony/Component/Form/Extension/Core/DataTransformer/DateTimeToStringTransformer.php

```diff
@@ -94,24 +94,25 @@ public function __construct($inputTimezone = null, $outputTimezone = null, $form
      *                                       instance or if the output timezone
      *                                       is not supported.
      */
-    public function transform($value)
+    public function transform($dateTime)
```

Comment:
@stof I changed it to make the argument name consistent across all five transformers. I can change it back, but I just need to understand how such code style clean-ups should be made instead? 1) As a separate patch for all branches in one go, 2) as a patch for master than is merged back, 3) such clean-ups are not worth the effort and should not be done at all, 4) something else?